### PR TITLE
Add aspnet/xdt to source-build.

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

**Comment by crummel** [comment]
I expect this to pass now.  Ready for review.

**Comment by crummel** [comment]
It builds Microsoft.Web.Xdt - I think it's used for XML transforms?

File: tools-local/prebuilt-baseline-offline.xml [hunk 1]
```diff
@@ -208,6 +196,7 @@
     <Usage Id="Microsoft.NETCore.Platforms" Version="2.1.2" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" IsDirectDependency="true" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies.net40" Version="1.0.0-alpha-5" />
+    <Usage Id="Microsoft.NETFramework.ReferenceAssemblies.net40" Version="1.0.0-preview.2" />
```
Comment (dseefeld) [inline tools-local/prebuilt-baseline-offline.xml]: I was expecting Microsoft.Web.Xdt.3.0.0 to go away as a prebuilt.

**Comment by vijayrkn** [comment]
> It builds Microsoft.Web.Xdt - I think it's used for XML transforms?

Yes this is used for Xml transforms. Primarily used in publish but the same technology is used in NuGet & App Service for Xml transformations

File: tools-local/prebuilt-baseline-offline.xml (hunk 1 above)
Comment (vijayrkn) [reply tools-local/prebuilt-baseline-offline.xml]: Xdt can be built from source. What needs to happen for this?

**Pull request merged**